client/src/snippets.json

```
@@ -183,5 +183,49 @@
             "</xrefs>"
         ],
         "description": "Add a xref section with bio.tools type."
-    }
+    },
+    "single output": {
+        "prefix": "gx-output-single",
+        "body": [
+            "<data name=\"--$1\" format=\"$2\" label=\"${tool.name} on ${on_string}\" />"
```

I guess those dashes were used for `argument` properties if I recall correctly so no need to have them on names.
```suggestion
            "<data name=\"$1\" format=\"$2\" label=\"${tool.name} on ${on_string}\" />"
```